ui/.env.production

```diff
@@ -0,0 +1 @@
+VUE_APP_API_URL=/
```

Comment:
By the way, I think they say it's an anti pattern to commit `.env` files: https://github.com/motdotla/dotenv#should-i-commit-my-env-file

Reply:
Is this env var needed at all now?

Reply:
Yes, it is used when building the APP. And we don't have anything secret in there.

Reply:
I don't mind keeping the variable in Dockerfile if you prefer

Reply:
Done